client/src/languageserver.ts

```diff
@@ -3,11 +3,54 @@ import {
 } from 'vscode-languageclient'
 import * as vscode from 'vscode';
 import * as net from 'net';
+import * as path from 'path';
 import * as cp from 'child_process';
 import ChildProcess = cp.ChildProcess;
 
 import * as messages from '../src/messages';
 
+export function createLanguageServerProcess() {
+
+  // "/some/file/structure/vscode-puppet/client"
+  var clientExtensionPath = vscode.extensions.getExtension("puppet.puppet-vscode").extensionPath;
+  // "/some/file/structure/vscode-puppet"
+  var extensionPath = path.dirname(clientExtensionPath);
+  // "/some/file/structure/vscode-puppet/server"
+  var serverPath = path.join(extensionPath, 'server');
+  // "/some/file/structure/vscode-puppet/server/puppet-languageserver"
+  var serverExe = path.join(serverPath, 'puppet-languageserver');
+
+  let cmd = '';
+  let args = [serverExe];
+  let options = {};
+
+  if (process.platform == 'win32') {
+    cmd = 'c:\\windows\\system32\\cmd.exe'
```

Comment:
Need to change this to sysnative on 64bit.  Vscode is a 32bit processs on Windows.  Should probably use %COMSPEC%  too?